plugins/available/cht-sh.plugin.bash

```diff
@@ -0,0 +1,9 @@
+cite about-plugin
+about-plugin 'Simplify `curl cht.sh/<query>` to `cht.sh <query>`'
+
+# Play nicely if user already installed cht.sh cli tool
+if ! _command_exists cht.sh ; then
+	function cht.sh () {
```

Comment:
Can you please add the `about`, `param`, `example`, and `group` meta function calls here?

Please see here for an example:

https://github.com/Bash-it/bash-it/blob/d9bcbf4c1fae2dd5c7b8090d544b600ec2a97b4f/plugins/available/base.plugin.bash#L92-L102

Reply:
Will do.

Q: Is there a `link` or `homepage` metadata element?  

Feels like it would be helpful to capture project homepages in the components that support 3rd party projects.

Reply:
Good question. These keywords are part of the composure framework: https://github.com/erichs/composure

The ones like `about` and `param` are part of the standard keywords.

You can define your own keywords using the `cite` function, e.g. `cite homepage`.

How that is then used is a different question. We're currently using these keywords in the `enabled` and `help` functions in Bash-it...